PWGLF/Tasks/rhoanalysis.cxx

```diff
@@ -0,0 +1,386 @@
+          continue;
+
+        pvec0 = std::array{_px1, _py1, _pz1};
+        pvec1 = std::array{_px2, _py2, _pz2};
+
+        auto arrMom = std::array{pvec0, pvec1};
+        mass = RecoDecay::m(arrMom, std::array{massPiplus, massPiminus});
+        pT = RecoDecay::pt(std::array{_px1 + _px2, _py1 + _py2});
+
+        rapidity = RecoDecay::y(std::array{_px1 + _px2, _py1 + _py2, _pz1 + _pz2}, mass);
+        if (std::abs(rapidity) >= 0.5)
+          continue;
+
+        histos.fill(HIST("hrholike"), mass, pT, multiplicity, rapidity);
+      }
+    }
+  }
+
+  PROCESS_SWITCH(rhoanalysis, processlike, "Process Same event", isLike);
+
+  ConfigurableAxis axisVertex{"axisVertex", {VARIABLE_WIDTH, -12, -10, -8, -6, -4, -2, 0, 2, 4, 6, 8, 10, 12}, "vertex axis for bin"};
+
+  ConfigurableAxis axisMultiplicity{"axisMultiplicity", {VARIABLE_WIDTH, 0.0, 2.75, 5.25, 7.75, 12.75, 17.75, 22.75, 27.75, 32.75, 37.75, 42.75, 47.75, 52.75, 57.75, 62.75, 67.75, 72.75, 77.75, 82.75, 87.75, 92.75, 97.75, 102.75, 107.75, 112.75, 117.75, 122.75, 127.75, 132.75, 137.75, 142.75, 147.75, 152.75, 157.75, 162.75, 167.75, 172.75, 177.75, 182.75, 187.75, 192.75, 197.75, 202.75, 207.75, 212.75, 217.75, 222.75, 227.75, 232.75, 237.75, 242.75, 247.75, 252.75, 257.75, 262.75, 267.75, 272.75, 277.75, 282.75, 287.75, 292.75, 297.75, 302.75, 307.75, 312.75, 317.75, 322.75, 327.75, 332.75, 337.75, 342.75, 347.75, 352.75, 357.75, 362.75, 367.75, 372.75, 377.75, 382.75, 387.75, 392.75, 397.75, 402.75, 407.75, 412.75, 417.75, 422.75, 427.75, 432.75, 437.75, 442.75, 447.75, 452.75, 457.75, 462.75, 467.75, 472.75, 477.75, 482.75, 487.75, 492.75, 497.75, 502.75, 507.75, 512.75, 517.75, 522.75, 527.75, 532.75, 537.75, 542.75, 547.75, 552.75, 557.75, 562.75, 567.75, 572.75, 577.75, 582.75, 587.75, 592.75, 597.75, 602.75, 607.75, 612.75, 617.75, 622.75, 627.75, 632.75, 637.75, 642.75, 647.75, 652.75, 657.75, 662.75, 667.75, 672.75, 677.75, 682.75, 687.75, 692.75, 697.75, 702.75, 707.75, 712.75, 717.75, 722.75, 727.75, 732.75, 737.75, 742.75, 747.75, 752.75, 757.75, 762.75, 767.75, 772.75, 777.75, 782.75, 787.75, 792.75, 797.75, 802.75, 807.75, 812.75, 817.75, 822.75, 827.75, 832.75, 837.75, 842.75, 847.75, 852.75, 857.75, 862.75, 867.75, 872.75, 877.75, 882.75, 887.75, 892.75, 897.75, 902.75, 907.75, 912.75, 917.75, 922.75, 927.75, 932.75, 937.75, 942.75, 947.75, 952.75, 957.75, 962.75, 967.75, 972.75, 977.75, 982.75, 987.75, 992.75, 997.75, 1002.75, 1007.75, 1012.75, 1017.75, 1022.75, 1027.75, 1032.75, 1037.75, 1042.75, 1047.75, 1052.75}, "multiplicity axis for histograms"};
+
+  using BinningType = ColumnBinningPolicy<aod::collision::PosZ, aod::mult::MultTPC>;
+  BinningType binningOnPositions{{axisVertex, axisMultiplicity}, true}; // true is for 'ignore overflows' (true by default)
+  SameKindPair<Event, TrackPi, BinningType> pair{binningOnPositions, cfgNoMixedEvents, -1, &cache};
+
+  void processMixedEvent(Event const& events, TrackPi const& tracks)
+  {
+
```

Comment:
```suggestion

  void processMixedEvent(Event const& events, TrackPi const& tracks)
  {
    BinningType binningOnPositions{{axisVertex, axisMultiplicity}, true}; // true is for 'ignore overflows' (true by default)
    SameKindPair<Event, TrackPi, BinningType> pair{binningOnPositions, cfgNoMixedEvents, -1, &cache};

```

Otherwise, your modified binning will not be affected when it is processed.